hubmap/query_app/validation.py

```diff
@@ -80,16 +84,21 @@ def recommend_identifiers(identifier: str, input_type: str):
             0:5
         ]
         similar_ids = set(similar_proteins.values_list("protein_id", flat=True))
+    elif input_type == "modality":
+        identifier_upper = identifier.upper()
+        annotated_modalities = Modality.objects.annotate(
+            similarity=TrigramSimilarity(Upper("modality_name"), identifier_upper)
+        )
+        similar_modalities = annotated_modalities.filter(similarity__gt=0.3).order_by(
```

Comment:
It might be nice to make these hardcoded values into constants defined at the top of the file or similar, but not a big deal.